src/Microsoft.AspNet.Mvc.Core/DefaultControllerFactory.cs

```diff
@@ -51,5 +74,93 @@ public void ReleaseController(object controller)
                 disposableController.Dispose();
             }
```

Comment:
The naming comes from @lodejard.

The factory is the contract for MVC, Activator is the implementation detail, so the release should stay here.